examples/sample-p5-app/src/index.js

```diff
@@ -31,9 +33,17 @@ const runCode = () => {
 
   outputDiv.innerHTML = '';
 
-  eval(code);
+  // Run p5 in instance mode. The name `sketch` matches the name
+  // used in the generator for all the p5 blocks.
+  // eslint-disable-next-line new-cap
+  new p5((sketch) => {
```

Comment:
I think you need to add `sketch` as a reserved word on the code generator so that you don't get conflicts with variable names.